src/app/room/components/room.component.html

```diff
@@ -1,3 +1,5 @@
+<p class="text-container">Let's find out who's gonna be the final  {{(room$ | async)?.typeOfGame === 1 ? 'loser' : 'winner'}}</p>
```

Comment:
I would prefer to use the old "enum in component trick" instead of the numbers to check here. That way, it is readable at least. `typeOfgame === 1` is a little obfuscating.

Reply:
You are right Monsieur, updated